android/src/main/java/io/twentysixty/rn/eidreader/EIdReaderModule.kt

```diff
@@ -273,6 +285,34 @@ class EIdReaderModule(reactContext: ReactApplicationContext) :
     }
   }
 
+  @ReactMethod
+  fun convertJp2000ToJpeg(data:String, promise: Promise){
+    try {
+      val dataSplit = data.split(";base64,")
+      val mimeType = dataSplit[0].split(":")[1]
+      if(mimeType != "image/jp2"){
```

Comment:
if (mimeType == "image/jpeg"), return the same contents, since we do not need to convert anything.

If mime type does not have an image, throw an error.